gatsby_fp/src/components/fields.js

```diff
@@ -0,0 +1,89 @@
+import React from "react"
+import GenInfoTable from "./fieldComponents/genInfoTable"
+import Education from "./fieldComponents/education"
+import ProfService from "./fieldComponents/profService"
+import UnivService from "./fieldComponents/univService"
+import ProfSocieties from "./fieldComponents/profSocieties"
+import IntellCont from "./fieldComponents/intellCont"
+import Section from "./fieldComponents/generalSection"
```

Comment:
We should probably import this as GeneralSection, to avoid possible conclusion with the HTML5 element `<section>`.

Reply:
See above comment